### ISSUE TITLE
VFX for stun state has text when mirrored

### ISSUE THREAD

**Issue by 6thmoon** [Goorakh/MirroredStageVariants#1]
not sure why but `EntityStates.StunState.stunVfxPrefab` seems to have an unused text element that only appears on mirrored stage variants

![stunned](https://github.com/Goorakh/MirroredStageVariants/assets/59669640/2fb8e662-037c-4699-81c5-da31c3844739)

pretty easy to fix like so but dunno if this is indicative of a different issue
```csharp
Addressables.LoadAssetAsync<GameObject>("RoR2/Base/Common/VFX/StunVfx.prefab").Completed
    += handle => handle.Result.GetComponentInChildren<TextMeshPro>().enabled = false;
```

**Comment by Goorakh** [comment]
I can't reproduce this issue, are you sure it's not another mod causing this?

**Comment by 6thmoon** [comment]
tried it on a completely fresh **r2modman** and in-game profile, still the same. thought it could be something to do with graphics settings but reset `config.cfg` too

could be hardware related; if nobody else is having the issue I'll just apply the fix locally

**Comment by 6thmoon** [comment]
https://www.twitch.tv/videos/2066904727?t=12m50s

**Comment by Goorakh** [comment]
Fixed #2
